example_package/doc/abczad.tex

```diff
@@ -1,13 +1,13 @@
 \documentclass[zad]{talentTex}
+
 \title{Przykładowe zadanie z długą nazwą}
 \id{abc}
-\iomode{stdin} % stdin / interactive
```

Comment:
?

Reply:
w kodzie jest `\iomode{stdin} % stdin`. Jak zaklepiemy interactive to poprawi się komentarz.